system/CodeIgniter.php

```diff
@@ -354,6 +361,30 @@ public function useSafeOutput(bool $safe = true)
         return $this;
     }
 
+    /**
+     * Invoked via spark command?
+     */
+    private function isSparked(): bool
+    {
+        return $this->context === 'spark';
+    }
+
+    /**
+     * Invoked via php-cli command?
+     */
+    private function isPhpCli(): bool
+    {
+        return $this->context === 'php-cli';
+    }
+
+    /**
+     * Web access?
+     */
+    private function isWeb(): bool
+    {
+        return $this->context === 'web';
+    }
+
```

Comment:
Unless there is a compelling reason for these methods I prefer the simple property check inline.

Reply:
I removed the private methods.